tests/functional/adapter/test_python_model.py

```diff
@@ -51,11 +47,10 @@ def test_changing_schema(self,project):
 from snowflake.snowpark.types import StructType, FloatType, StringType, StructField
 
 def model( dbt, session):
-    
+
     dbt.config(
         materialized='table',
         imports = ['@dbt_integration_test/iris.csv'],
-        use_anonymous_sproc = True
```

Comment:
Maybe modify this to be false so the we have one test testing the other path

Reply:
was wanting to double check with you on that, if we had a specific one we preferred flipping or if we just wanted to add another test like the initial one that swaps there. will make addition